### PR TITLE
[BugFix] Stop silent failures on compressed-tensors parsing

### PR DESCRIPTION
**Pull request by dsikka**
# Summary
- compressed-tensors has been silently failing if there are any config parsing errors and just sets the `input_activations` to None in those cases, which results in downstream failures that hide the root problem
- PR updates to stop the silent failure and changes the condition on `input_quant`

### CODE REVIEW

File: vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py [hunk 1]
```diff
@@ -97,11 +97,12 @@ def from_config(cls, config: Dict[str, Any]) -> "CompressedTensorsConfig":
                 target_scheme_map[target][
                     "weights"] = QuantizationArgs.parse_obj(
                         quant_config.get("weights"))
-                try:
+
+                if is_activation_quantization_format(quant_format):
                     target_scheme_map[target][
                         "input_activations"] = QuantizationArgs.parse_obj(
                             quant_config.get("input_activations"))
-                except Exception:
```
Comment (rahul-tuli) [inline vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py:104]: nice!

**Review by rahul-tuli — APPROVED** [review]

**Review by mgoin — APPROVED** [review]

**Comment by mgoin** [comment]
@dsikka I think there is a real issue in the CI here https://buildkite.com/vllm/ci-aws/builds/9886#0192911e-1065-4a85-8a1b-75071e80f1de/193-715

**Comment by dsikka** [comment]
> @dsikka I think there is a real issue in the CI here https://buildkite.com/vllm/ci-aws/builds/9886#0192911e-1065-4a85-8a1b-75071e80f1de/193-715

@mgoin Yup, which is why we need the PR which adds compressed-tensors to land. The current utils is out-of-date. This PR is secondary to make future failures more visible/dont hide the root cause. 

See ticket:
https://app.asana.com/0/0/1208535979523227/1208536852343502/f

**Review by mgoin — APPROVED** [review]
🤞

**Pull request merged**